.gitignore

```diff
@@ -0,0 +1,49 @@
+### Emacs ###
```

Comment:
These are all files that are relevant to you privately, not generally.

You can git-ignore files privately by putting this into `.git/info/exclude`.